iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java

```diff
@@ -0,0 +1,218 @@
+package com.dataloom.integrations.iowacity;
+
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+
+public class Nemesis {
```

Comment:
It's good practice to set a private constructor in classes like these where everything is just a constant:

```
public class Nemesis {
  private Nemesis() {}
}
```